index.js

```diff
@@ -1,5 +1,69 @@
 'use strict';
 
+var superagent = require('superagent');
+
+global.D2LAccessTokenExpiresAt = 0;
+
+function now() {
+	return Date.now()/1000 | 0;
+}
+
+function addHeaders(req) {
+	req.set('X-Csrf-Token', localStorage['XSRF.Token']);
```

Comment:
Hm, http://stackoverflow.com/a/15439059 . Not really sure if null or undefined is preferable. Do you have an opinion?
Now that you mention it logging something to the console when we can't get the xsrf token would probably be helpful.